ruby/example_code/cloudtrail/README.md

```diff
@@ -1,61 +1,56 @@
-# AWS SDK for Ruby code examples for AWS CloudTrail
+# CloudTrail code examples for the AWS SDK for Ruby
```

Comment:
The template uses the short SDK name in the title, which is correct. However, the README files throughout show titles  with the long SDK names. Instead of "AWS SDK for Ruby", it's "SDK for Ruby".